packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js

```diff
@@ -12,6 +12,7 @@ export default class OverTheBoxDetailsCtrl {
     $filter,
     $translate,
     $q,
+    coreConfig,
```

Comment:
Looks like unused but might be useful if you want to access to `ovhSubsidiary`. Isn't it?

Reply:
For the moment, there is no use of ovhSubsidiary for OverTheBox. I remove this definition